CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing Guidelines
+
+First off, **thank you** for considering contributing to this project!
+
+## Table of contents
+
+- [I Have a Question](#i-have-a-question)
+- [I Want to Contribute](#i-want-to-contribute)
+
+## <a name="i-have-a-question">I Have a Question</a>
+
+See our [support guidelines](https://github.com/safe-global/safe-core-sdk/tree/main/SUPPORT.md). **Do not** use GitHub issues for general support or questions.
+
+## <a name="i-want-to-contribute">I Want to Contribute</a>
+### Legal Notice
+You will need to agree to [our CLA](https://safe.global/cla) in order to be possible to consider your contribution.
```

Comment:
Should we explain how to agree ?

Reply:
I think is enough with the bot instructions. If it wasn't enough then it shold be better to update the action to print a longer message. Here it will just add complexity to the document, while I thought it was important for them to know they should agree to a CLA before doing the effort to create the code